### PR TITLE
[16.0][MIG] stock_cycle_count

### PR DESCRIPTION
**Pull request by Urvisha-OSI**

### CODE REVIEW

**Comment by rousseldenis** [comment]
@Urvisha-OSI To allow reviewers being able to test this, could you include your dependencies following: https://github.com/OCA/maintainer-tools/wiki/Use-temporary-reference%28s%29-to-another-pull-request%28s%29 ?

Thanks

**Comment by serCliff** [comment]
Hello @Urvisha-OSI could you make a rebase and fix the test?

After that I could try the module with the runbot.


Thank you

File: stock_cycle_count/models/stock_inventory.py [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+
+class StockInventory(models.Model):
+    _inherit = "stock.inventory"
+
+    @api.depends("state", "stock_quant_ids")
+    def _compute_inventory_accuracy(self):
+        for inv in self:
+            theoretical = sum(inv.stock_quant_ids.mapped(lambda x: abs(x.quantity)))
+            abs_discrepancy = sum(
+                inv.stock_quant_ids.mapped(lambda x: abs(x.discrepancy_qty))
```
Comment (ArnauCForgeFlow) [inline stock_cycle_count/models/stock_inventory.py:19]:
```suggestion
                inv.stock_quant_ids.mapped(lambda x: abs(x.inventory_diff_quantity))
```
"discrepancy_qty" it seems that is no longer available in version 16, i suggest change this attribute for "inventory_diff_quantity"

**Review by ArnauCForgeFlow — CHANGES_REQUESTED** [review]

**Comment by rousseldenis** [comment]
@Urvisha-OSI Could you update this ?

**Comment by LoisRForgeFlow** [comment]
Closed due to inactivity in favor of https://github.com/OCA/stock-logistics-warehouse/pull/2045

**Pull request closed without merging**